spec/features/user_signs_up_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-feature 'user registers', %Q{
+feature 'user registers', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.